app/views/browse/stateful-set.html

```diff
@@ -0,0 +1,178 @@
+<project-header class="top-header"></project-header>
+<project-page>
+  <div class="middle-section">
+    <div class="middle-container">
+
+      <div class="middle-header">
+        <div class="container-fluid">
+          <breadcrumbs breadcrumbs="breadcrumbs"></breadcrumbs>
+          <alerts alerts="alerts"></alerts>
+          <div>
+            <h1>
+              {{statefulSet.metadata.name}}
+              <div
+                class="pull-right dropdown"
+                ng-if="statefulSet"
+                ng-show="resourceGroupVersion.resource | canIDoAny">
+                <button
+                  type="button"
+                  class="dropdown-toggle btn btn-default actions-dropdown-btn hidden-xs"
+                  data-toggle="dropdown">
+                  Actions
+                  <span class="caret" aria-hidden="true"></span>
+                </button>
+                <a
+                  href=""
+                  class="dropdown-toggle actions-dropdown-kebab visible-xs-inline"
+                  data-toggle="dropdown">
+                  <i class="fa fa-ellipsis-v"></i><span class="sr-only">Actions</span>
+                </a>
+                <ul class="dropdown-menu actions action-button">
+                  <!-- TODO: kinda works, but cant seem to edit. neat. -->
+                  <li ng-if="resourceGroupVersion | canI : 'update'">
+                    <a ng-href="{{statefulSet | editYamlURL}}" role="button">Edit YAML</a>
+                  </li>
+                  <li ng-if="resourceGroupVersion | canI : 'delete'">
+                    <!-- TODO: Warn about replicas? deleting statefulSets doesn't delete the pods -->
+                    <!-- TODO: hpa-list attrib? figure out autoscaling -->
```

Comment:
autoscaling can be a follow-on but add a task to the trello story so QE can know what is / isnt finished